submissions/adeyemiJuwon/css/product.css

```diff
@@ -0,0 +1,307 @@
+footer{
+    background-color:rgb(255, 166, 95) ;
+}
+.footer-link{
+    display: flex;
+    justify-content: space-evenly;
+    padding-top: 37px;
+    flex-wrap: wrap;
+
+}
+.footer-link ul {
+    list-style: none;
+}
+.footer-link ul li{
+    color: white;
+    line-height: 22px;
+    font-size: 18px;
+    margin-bottom: 15px;
+}
+.footer-link ul a{
+    text-decoration: none;
+    color: white;
+}
+.footer-link input{
+    width: 253px;
+    height: 44px;
+    line-height: 19px;
+    font-size: 16px;
+    color: rgb(255, 166, 95) ;
+}
+.footer-link button{
+    width: 108px;
+    height: 44px;
+    font-size:18px;
+    line-height: 22px;
+    margin-top: 9px;
+    color: rgb(255, 166, 95) ;;
+}
+footer p{
+text-align: center;
+color: #fff;
+}
+.social-link{
+    margin-top: 13px;
+    margin-left: auto;
+}
+.social h3, .footer-link h3{
+    color: white;
+}
+.social-link li{
+    display: inline-block;
+    margin-left: 12px;
+}
+
+/**********************************
+  RESPONSIVENESS
+  ***********************************/
+
+
+@media screen and ( max-width:1085px ){
+
+    
+/**********************************
+  HEADING
+  ***********************************/
+    header{
+        flex-flow:wrap;
+        justify-content: flex-end;
+    
+    }
+    .navbar-link li{
+        margin: 15px;
+    }
+    
+/**********************************
+  PAGE:PRODUCT
+  ***********************************/
+.product-img{
+    margin: 10px;
+}
+.product-del button{
+    width: 130px;
+    font-size: 18px;
+    line-height: 20px;
+}
+.description h3, .description p{
+    font-size: 14px;
+    line-height: 18px;
+}
+.description p{
+    margin: 10px;
+}
+.product-del h3, .product-del h4{
+    font-size: 20px;
+    line-height: 24px;
+}
+.lap-1, .charger, .bag{
+    flex-basis: 250px;
+    margin: 20px;
+}
+.product-related{
+    justify-content: space-between;
+    margin-top: 20px;
+}
+}
+
+@media screen and ( max-width:600px ) {
+
+
+/**********************************
+  HEADER
+***********************************/
+header{
+    flex-direction: column;
+    align-items: flex-start;
+}
+.navbar-link li {
+    display: block;
+    text-align: left;
+    line-height: 19px;
+    font-size: 16px;
+    margin-left:10px ;
+    color:black;
+}
+.logo{
+    margin-left: 10px;
+}
+/**********************************
+PAGE:PRODUCT
+***********************************/
+
+.product{
+    flex-wrap: wrap;
+    margin: 20px;
+}
+.product-del{
+    margin: 20px;
+}
+.description{
+    margin-left: 30px;
+}
+.product-related{
+    justify-content: space-evenly;
+    margin: 10px;
+}
+.related-product h3 {
+    font-size: 20px;
+}
+.lap-1, .charger, .bag{
+    flex-basis: 399px;
+    margin: 10px;
+}
+/**********************************
+FOOTER
+***********************************/
+.footer-link{
+    display: block;
+    margin-left: 15px;
+}
+.social h3{
+    margin-top: 20px;
+}
+.social-link li{
+    margin-right: 5px;
+}
+    
+}
```

Comment:
@Samju200 Which opening tag is this attached to?

Reply:
social-link li tag

Reply:
The last closing tag is for the overall tag for media query max-length 600px screen.